### PR TITLE
This is for gunrock issues #69

### PR DESCRIPTION
**Pull request by wetliu**
Has changed the if branch for the bogus file name and added the exit part. Not sure yet if exit is the best because a lot of people say that exit is not as good as return, but using return has to change main().

### CODE REVIEW

**Comment by jowens** [comment]
@yzhwang @wetliu Shouldn't this be a merge into dev not master?

File: gunrock/app/enactor_base.cuh [hunk 1]
```diff
@@ -646,11 +646,14 @@ public:
...
                 printf("Loading Matrix-market coordinate-formatted graph ...\n");
             }
+
             char *market_filename = args.GetCmdLineArgvDataset();
-            if (market_filename == NULL)
+
+            std::ifstream fp(market_filename);
+            if (market_filename == NULL||!fp.is_open())
             {
                 fprintf(stderr, "Input graph does not exist.\n");
```
Comment (jowens) [inline gunrock/app/enactor_base.cuh]: I would print the filename here too.

Comment (yzhwang) [inline gunrock/app/enactor_base.cuh]: I agree with @jowens on adding the filename to stderr. Have you run ctest after the change? Also, please test for all primitives for a bogus filename or an empty filename, and see if the output make sense. (You can post the output here, you can and you should write a simple shell/python script for your test too)

As for pushing to the master, @wetliu I might have told you to push to the master, if so I apologize, according to https://github.com/gunrock/gunrock/issues/85 master is for actual release. So please cherrypick this commit to the dev branch of your fork and do the things I suggested, then submit the pull request again. Thank you!

**Comment by wetliu** [comment]
I am sorry that it's my mistake to merge it to master. I can print a file name there. I have not yet written a shell script. I will do it later. Thank you for your comments @jowens @yzhwang
